lighthouse-core/audits/audit.js

```diff
@@ -57,6 +57,60 @@ class Audit {
   }
 
   /**
+   * @param  {!Audit.Headings} headings
+   * @return {*}
+   */
+  static makeV1TableHeadings(headings) {
+    const tableHeadings = {};
+    headings.forEach(heading => tableHeadings[heading.key] = heading.text);
+    return tableHeadings;
+  }
+
+  /**
+   * @param  {!Audit.Headings} headings
+   * @param  {!Object} results
+   * @return {!Array<!DetailsRenderer.DetailsJSON>}
+   */
+  static makeV2TableRows(headings, results) {
+    const tableRows = results.map(item => {
+      return headings.map(heading => {
+        return {
+          type: heading.itemType,
```

Comment:
shrugz

I'm okay with either.
patrick is fine with either but thinks its slightly better to have all things coming into renderer be our details objects
dgozman also feels the same way, and thinks that if we wanted to conserve the size of the LH object we should shorten `type` to `t` etc. ;) ;) 

so I'm comfortable expanding the objects over here before we deliver the LHR.